pkg/aws/actuator/actuator_test.go

```diff
@@ -446,9 +446,9 @@ func TestSecretFormat(t *testing.T) {
 		t.Run(test.name, func(t *testing.T) {
 			var fakeClient client.Client
 			if test.existingSecret != nil {
-				fakeClient = fake.NewFakeClient(test.existingSecret)
+				fakeClient = fake.NewClientBuilder().WithRuntimeObjects(test.existingSecret).Build()
 			} else {
-				fakeClient = fake.NewFakeClient()
+				fakeClient = fake.NewClientBuilder().WithRuntimeObjects().Build()
```

Comment:
Is 
```suggestion
				fakeClient = fake.NewClientBuilder().Build()
```
a legitimate way to spell this?

Reply:
Good catch, updated to remove redundant builder function call